scripts/us_epa/superfund/sites/tar_creek/process_contaminants.py

```diff
@@ -0,0 +1,240 @@
+    sv_list = ['Node: ' + x for x in sv_list]
+    f = open(output_path, 'w')
+    for k, pvs in sv_dict.items():
+        if k in sv_list:
+            fstr = k + "\n"
+            for p, v in pvs.items():
+                if v.startswith('dcs:'):
+                    fstr += f"{p}: {v}\n"
+                else:
+                    fstr += f"{p}: dcs:{v}\n"
+            fstr += "\n"
+            f.write(fstr)
+    f.close()
+
+
+def clean_dataset(row: pd.Series, column_list: list, sv_map: dict) -> None:
+    clean_csv = pd.DataFrame(columns=[
+        'observationAbout', 'observationDate', 'variableMeasured', 'value',
+        'units'
+    ])
+
+    ignore_list = ['-', ' - ', 'NaN', 'n.a.', np.nan]
+    ignore_columns = ['observationAbout', 'observationDate', 'contaminantType']
+
+    for column in column_list:
+        if column not in ignore_columns:
+            if row[column] not in ignore_list:
+                sv_key = column + '_' + row['contaminantType']
+
+                ## add data to clean csv
+                clean_csv = clean_csv.append(
+                    {
+                        'observationAbout': row['observationAbout'],
+                        'observationDate': row['observationDate'],
+                        'variableMeasured': 'dcid:' + sv_map[sv_key],
+                        'value': row[column],
+                        'units': _UNIT_MAP[column]
+                    },
+                    ignore_index=True)
+
+    ## append to a list of dataframes
+    _CLEAN_CSV_FRAMES.append(clean_csv)
+
+
+def process_intermediate_csv(input_path: str, output_path: str) -> None:
+    ## Create output directory if not present
+    if not os.path.exists(output_path):
+        os.makedirs(output_path)
+
+    df = pd.read_csv(os.path.join(input_path, _INTERMEDIATE_CSV_NAME))
+    df.replace(to_replace=r'^<.*$', value=0, regex=True,
+               inplace=True)  #remove < from SVObs values
+
+    contaminant_list = df['contaminantType'].unique().tolist()
+    sv_dict, sv_column_map = make_stat_var_dict(contaminant_list,
+                                                df.columns.tolist())
+
+    df.apply(clean_dataset, args=(
+        df.columns.tolist(),
+        sv_column_map,
+    ), axis=1)
+
+    clean_csv = pd.concat(_CLEAN_CSV_FRAMES, ignore_index=True)
+    clean_csv = clean_csv[~(clean_csv['value'] == '.') &
+                          ~(clean_csv['value'] == 'na') &
+                          ~(clean_csv['value'] == 'n .a .')]
+    clean_csv['value'] = clean_csv['value'].astype(float)
+    # there are multiple samples taken during the same day from the same well. For these cases, we aggregate the values to the maxValue
+    clean_csv['value'] = clean_csv.groupby(
+        ['observationAbout', 'observationDate', 'variableMeasured', 'units'],
+        as_index=False)['value'].transform(max)
+
+    write_sv_mcf(os.path.join(output_path, "superfund_sites_tarcreek.mcf"),
```

Comment:
use a constant for superfund_sites_tarcreek.*(or use an argument / flag for this)

Reply:
Thanks for the suggestion, I have used a constant for this now.